cmd/svcat/touch/touch_cmd.go

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package touch
+
+import (
+	"github.com/kubernetes-incubator/service-catalog/cmd/svcat/command"
+	"github.com/spf13/cobra"
+)
+
+// NewCmd builds a "svcat touch instance" command
```

Comment:
nit: since this touch subcommand isn't specific to an instance (it's just `svcat touch` unlike the next file which is `svcat touch instance`), we should use the generic term resource instead. Example below:

https://github.com/kubernetes-incubator/service-catalog/blob/01e652ff75a7b418f23d8173cc7191e986d16a77/cmd/svcat/main.go#L126-L131